tests/test_core_schedule.py

```diff
@@ -1295,6 +1295,23 @@ def test_generating_new_vehicles_with_overwite_True(schedule):
     assert_semantically_equal(schedule.vehicles, {'veh_3_bus': {'type': 'bus'}, 'veh_4_bus': {'type': 'bus'},
                                                   'veh_1_bus': {'type': 'bus'}, 'veh_2_bus': {'type': 'bus'}})
 
+def test_scale_vehicle_capacity(schedule, tmpdir):
```

Comment:
Could you add one more test : `test_scaling_vehicle_capacity_does_not_affect_original_values` - or named similarly :D a test where you assert what the numbers are, make a scaling change and then assert the numbers attached to the vehicles in the schedule are still the same - that way you can apply this function a number of times to generate 1 and 10 and other percentage vehicles making sure that it's not compounded :D

Reply:
Just did, thank you Kasia for the reminder :)